products/dns/terraform.yaml

```diff
@@ -89,6 +89,7 @@ overrides: !ruby/object:Overrides::ResourceOverrides
           and apply an incorrect update to the resource. If you encounter this issue, remove all `networks`
           blocks in an update and then apply another update adding all of them back simultaneously.
       privateVisibilityConfig.networks.networkUrl: !ruby/object:Overrides::Terraform::PropertyOverride
```

Comment:
Do you also want to update the docs for these attributes if they say they have to be formatted as the full URL?